src/executables/csql.c

```diff
@@ -1460,6 +1464,205 @@ csql_do_session_cmd (char *line_read, CSQL_ARGUMENT * csql_arg)
 	}
 
       break;
+
+    case S_CMD_CONNECT:
+      /*dbname_cpy can be stored DB name+ @ + remote hostname + \0 */
+      char dbname_cpy[MAXLOGNAME + CUB_MAXHOSTNAMELEN + 2];
+      char username_cpy[DB_MAX_USER_LENGTH + 1];
+      char host_cpy[DB_MAX_USER_LENGTH + 1];
```

Comment:
I don't feel comfortable with variable names of style **username_cpy**. What about following
* without dash: array name
* with dash: pointer name

as char array
username [DB_MAX_USER_LENGTH + 1]]
hostname [DB_MAX_USER_LENGTH + 1]
dbname [MAXLOGNAME + CUB_MAXHOSTNAMELEN + 2]]

and as a char pointer (as you used)
char *user_name;
char *db_name;
char *host_name;

Reply:
Could we  change ***user_name**;  as char ***user_name_ptr**?

Reply:
Sure, **user_name_ptr** style is good.

Reply:
@kisoo-han 
Thank you so much!